app-crypt/tpm2-tss/tpm2-tss-2.3.3.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info udev
+
+DESCRIPTION="TCG Trusted Platform Module 2.0 Software Stack"
+HOMEPAGE="https://github.com/tpm2-software/tpm2-tss"
+SRC_URI="https://github.com/tpm2-software/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0/0"	# sublot is libtss2-sys number
+KEYWORDS="~amd64"
+IUSE="doc +gcrypt openssl static-libs test"
+
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="
+	gcrypt? ( !openssl )
+	openssl? ( !gcrypt )
+	|| ( gcrypt openssl )"
```

Comment:
So this logic says either one of gcrypt or openssl must be selected? You can use `^^` instead of `||` to force it choose exactly-one-of and remove two lines above.